ibm_quantum_schemas/models/__init__.py

```diff
@@ -10,4 +10,4 @@
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
 
-"""Models and validation for the Noise Learner V3."""
+"""Models and validation for types used by executor."""
```

Comment:
Good catch, but not just for executor
```suggestion
"""Models"""
```